common/src/main/java/com/wynntils/features/inventory/ItemScreenshotFeature.java

```diff
@@ -141,6 +142,10 @@ private void takeScreenshot(Screen screen, Slot hoveredSlot, List<Component> ite
 
         BufferedImage bi = SystemUtils.createScreenshot(fb);
 
+        // Free the buffer source to prevent memory leaks
+        byteBuffer.close();
```

Comment:
If the buffer implements Closeable you can use try-with-resources.

Reply:
Yeah, I'd have to check, it might not, I'd have expected the BufferSource itself to be closable, but it is not.